forge-gui/res/cardsfolder/upcoming/skoa_embermage.txt

```diff
@@ -0,0 +1,9 @@
+Name:Skoa, Embermage
+ManaCost:4 R R
+Types:Legendary Creature Goblin Wizard
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDealDamage | TriggerDescription$ When CARDNAME enters the battlefield, it deals 4 damage to any target.
+SVar:TrigDealDamage:DB$ DealDamage | ValidTgts$ Any | NumDmg$ 4
+A:AB$ DealDamage | Cost$ Discard<1/Card.namedSkoa, Embermage> Sac<2/Mountain> | CostDesc$ Grandeur — Discard another card named Skoa, Embermage, Sacrifice two Mountains: | ValidTgts$ Any | NumDmg$ 4 | SpellDescription$ NICKNAME deals 4 damage to any target.
```

Comment:
@Simisays 
did you check if your change ; ➡️  , works?
The valid might get split badly with it now

Reply:
I think so yeah 
![image](https://github.com/Card-Forge/forge/assets/67333662/8f9a0ff0-10d9-42fb-9f7c-4062738d2075)

The previous version didn't let me activate even though all the conditions where met